--Esta nueva entrega del proyecto se debe a que hemos tenido que actualizar, ambos miembros de la pareja, nustra version de Django.

--En el fichero tests.py nos hemos visto obligados a comentar el test 'PageAccessTests' dado que no hemos sido capaces de implementarlo de foma correcta.

--El fichero 'populate.py' se ha llamado en este proyecto 'populate_rango.py'

